Domains/2-DisciplinePhysical/StructuralPhysical/StructuralPhysical.ecschema.xml

```diff
@@ -34,31 +34,139 @@
     <ECEntityClass typeName="Slab" displayLabel="Slab">
         <BaseClass>StructuralMember</BaseClass>
     </ECEntityClass>
+    <ECEntityClass typeName="SlabType" displayLabel="Slab Type">
+        <BaseClass>bis:PhysicalType</BaseClass>
+    </ECEntityClass>
+    <ECRelationshipClass typeName="SlabIsOfType" strength="referencing" modifier="None" description="A type-instance relation; one that indicates that the specific sp:Slab is an instance of the defined sp:SlabType.">
```

Comment:
Could we reuse `bis:PhysicalElementIsOfType` relationship for mapping newPhysicalType to PhysicalElements instead?
What advantages do you see of having separate relationships for each type?

Reply:
These sub-classes of bis:PhysicalElementIsOfType are primarily for documentation purposes. That is, it is easier for consumers of the schema to realize what PhysicalType-subclass is meant to be associated with what PhysicalElement-subclass.